Sources/Apodini/Components/Component/Component.swift

```diff
@@ -54,6 +54,14 @@ private protocol HandlerVisitorHelperImplBase: AssociatedTypeRequirementsVisitor
     func callAsFunction<H: Handler>(_ value: H) -> Output
 }
 
+extension HandlerVisitorHelperImplBase {
+    @inline(never)
+    @_optimize(none)
+    func _test() {
+        _ = self(Text(""))
```

Comment:
This method is never called ;) 
It's just a workaround to make sure the compiler doesn't delete the method during optimizations...

So the way this works is that `AssociatedTypeRequirementsVisitor` requires a `_test` function where you're supposed to call `callAsFunction`. 
By adding it as an extension to the protocol in the middle we take away the work from the implementations. And since `callAsFunction` is being called from a protocol extension, all implementations of it will be marked as if it's possible that they will be called at some point